smart-cards/src/main/java/gov/va/api/health/smartcards/vc/CredentialType.java

```diff
@@ -4,10 +4,13 @@
 import java.util.Arrays;
 
 public enum CredentialType {
-  COVID_19("https://smarthealth.cards#covid19"),
+  // Purposely NOT alphabetized!
+  // These types become more granular the lower they are in this list.
+  // We want to keep the enum's natural order (by order of declaration) when sorting!
```

Comment:
I think these comments can simplify to something like `// ordered by granularity for sorting`